src/raftstore/store/fsm/peer.rs

```diff
@@ -789,6 +792,28 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
             self.fsm.has_ready = true;
         }
 
+        // Send a read index to renew heartbeat
+        if self.fsm.peer.is_leader() {
+            let last_pending_read_count = self.fsm.peer.raft_group.raft.pending_read_count();
+            let last_ready_read_count = self.fsm.peer.raft_group.raft.ready_read_count();
+
+            let id = Uuid::new_v4();
```

Comment:
Seems for every base tick the leader will bcast heartbeats to renew lease?

Reply:
Thanks for reminding me. it's just for test now, I'll change it later.